### DISCUSSION TITLE
Webpage display or app display

### DISCUSSION THREAD

**Discussion by Wulfman24** [AntonyCorbett/OnlyM#427] [Ideas]
Hello 👋
Silly question. I'm having a fun time try to setup first screen controls and second screen video output only with VLC. Tried everything. Any chance you can add the ability to display a webpage or app?

Only reason we still use VLC at convention is because of chrome and live stream. But being able to do that with onlyM and using second screen output would be beautiful!! VLC is needlessly overcomplicated. And frequently doesn't do what it says it can.

**Comment by AntonyCorbett** [comment]
@Wulfman24 OnlyM can display web content (see the Wiki under **Operation => Websites**)
